frontend/src/v5/ui/routes/dashboard/projects/federations/federationsList/federationListItem/federationListItem.component.tsx

```diff
@@ -62,11 +62,12 @@ export const FederationListItem = ({
 	filterQuery,
 	onFavouriteChange,
 }: IFederationListItem): JSX.Element => {
-	const { teamspace, project } = useParams<DashboardParams>();
-
 	if (federation.hasStatsPending) {
 		return <SkeletonListItem delay={index / 10} key={federation._id} />;
 	}
+
+	const { teamspace, project } = useParams<DashboardParams>();
```

Comment:
![image](https://user-images.githubusercontent.com/43547703/182584214-964b8b36-daad-436a-82e8-0d6514f283f6.png)

This is related to the `list of hooks`(_lof_ for simplicity) and its length.

Before the change, when the federation had pending stats, the length of _lof_ was 1, otherwise it was 2. So doing, React detected a change in its length.

Moving the position of `if (federation.hasStatsPending) { ... }` to the top, when the condition is true, _lof_ does not exist as no hooks is ever called, so react does not detect a change in its length when the if condition becomes false

> For reference: https://stackoverflow.com/questions/57397395/react-has-detected-a-change-in-the-order-of-hooks-but-hooks-seem-to-be-invoked